languages/co-authors-plus-he_IL.po

```diff
@@ -0,0 +1,593 @@
+# Copyright (C) 2014 Co-Authors Plus
+# This file is distributed under the same license as the Co-Authors Plus package.
+msgid ""
+msgstr ""
+"Project-Id-Version: Co-Authors Plus 3.1-beta\n"
+"Report-Msgid-Bugs-To: http://wordpress.org/tag/co-authors-plus\n"
+"POT-Creation-Date: 2014-03-17 15:59:18+00:00\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2019-01-07 16:19+0200\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.1.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : n==2 ? 1 : n>10 && n%10==0 ? "
+"2 : 3);\n"
+"Language: he_IL\n"
+
+#: co-authors-plus.php:300 co-authors-plus.php:401 co-authors-plus.php:487
+#: co-authors-plus.php:1290
+msgid "Authors"
+msgstr "מחברים"
+
+#: co-authors-plus.php:362
+msgid ""
+"<strong>Note:</strong> To edit post authors, please enable javascript or use "
+"a javascript-capable browser"
+msgstr ""
+"<strong>שימו לב:</strong> כדי לערוך את מחברי הפוסט יש לאפשר שימוש "
+"בJavascript בדפדפן."
+
+#: co-authors-plus.php:369 co-authors-plus.php:489 co-authors-plus.php:1095
+msgid ""
+"Click on an author to change them. Drag to change their order. Click on "
+"<strong>Remove</strong> to remove them."
+msgstr ""
+"לחצו על מחבר כדי לשנות אותו. גררו מחברים כדי לשנות את הסדר שלהם. לחצו על "
+"<strong>הסרה</strong> כדי למחוק אותם."
+
+#: co-authors-plus.php:449 php/class-coauthors-wp-list-table.php:148
+msgid "Posts"
+msgstr "פוסטים"
+
+#: co-authors-plus.php:466
+msgid "View posts by this author"
+msgstr "הצגת פוסטים לפי מחבר"
+
+#: co-authors-plus.php:531 co-authors-plus.php:548
+msgid "No co-author exists for that term"
+msgstr "לא נמצאו מחברים"
+
+#: co-authors-plus.php:1090 php/class-coauthors-wp-list-table.php:212
+msgid "Edit"
+msgstr "עריכה"
+
+#: co-authors-plus.php:1091
+msgid "Remove"
+msgstr "הסרה"
+
+#: co-authors-plus.php:1092
+msgid "Are you sure you want to remove this author?"
+msgstr "האם באמת להסיר את המחבר הזה?"
+
+#: co-authors-plus.php:1093
+msgid "Click to change this author, or drag to change their position"
+msgstr "לחצו לעריכת המחבר, או גררו לשינוי המיקום שלו"
+
+#: co-authors-plus.php:1094
+msgid "Search for an author"
+msgstr "חיפוש מחברים"
+
+#: co-authors-plus.php:1132
+msgid "Mine"
+msgstr "שלי"
+
+#: co-authors-plus.php:1402
+msgid "New comment on your post \"%s\""
+msgstr "תגובות חדשות על הפוסט שלך \"%s\""
+
+#. translators: 1: comment author, 2: author IP, 3: author domain
+#: co-authors-plus.php:1404 co-authors-plus.php:1521
+msgid "Author : %1$s (IP: %2$s , %3$s)"
+msgstr "מחבר : %1$s (IP: %2$s , %3$s)"
```

Comment:
The trailing s assoc with %1$s has been added to the start of the translation here. This happens a number of times below because of the LTR alignment. I think you're probably aware of this but just in case I wanted to highlight it.

Reply:
Hi @Nic-Sevic 👋 I haven't noticed this. I'll go over the file and correct it directly.

Reply:
I just checked the file using [Poedit](https://poedit.net/), and it seems that GitHub cannot display the placeholders correctly. When checking the file with Poedit, the placeholder appears as expected.